coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -154,6 +154,12 @@ def forward(self, x):
 
         self.run_compare_torch(model.input_size, model, backend=backend, use_scripting=True)
 
+
```

Comment:
not seeing why we are adding those lines. Please remove it.

Reply:
Done